python/private/bzlmod/BUILD.bazel

```diff
@@ -32,19 +32,26 @@ bzl_library(
         ":pip_repository_bzl",
         "//python/pip_install:pip_repository_bzl",
         "//python/pip_install:requirements_parser_bzl",
+        "//python/private:pypi_index_bzl",
         "//python/private:full_version_bzl",
         "//python/private:normalize_name_bzl",
         "//python/private:parse_whl_name_bzl",
         "//python/private:version_label_bzl",
         ":bazel_features_bzl",
+        ":platforms_host_bzl",
     ] + [
         "@pythons_hub//:interpreters_bzl",
     ] if BZLMOD_ENABLED else [],
 )
 
 bzl_library(
     name = "bazel_features_bzl",
-    srcs = ["@bazel_features//:bzl_files"] if BZLMOD_ENABLED else [],
+    srcs = ["@bazel_features//:bzl_files"],
+)
+
+bzl_library(
+    name = "platforms_host_bzl",
+    srcs = ["@platforms//host:srcs"] if BZLMOD_ENABLED else [],
```

Comment:
as a stopgap, you could list the .bzl files here explicitly (before https://github.com/bazelbuild/platforms/pull/89) is discussed and merged

Reply:
@Wyverald, if I understand correctly, I can't do it. This is because the host platform extension is internal to the platforms module. The extension that creates it is used internally by platforms and I am not sure I should use the extession myself in rules python. Or maybe it is exactly how I should consume this code? Please advise. :)

Reply:
Anyway, I decided to not use `platforms` in the repository context, the PRs are now decoupled, we can discuss it in bazelbuild/platforms#89.